tests/containers/rootless_podman.pm

```diff
@@ -63,7 +64,9 @@ sub run {
 }
 
 sub softfail_and_skip_on_bsc1182874 {
-    if (script_run("podman run alpine", timeout => 120) != 0) {
+    my $alpine = registry_url('alpine');
+    script_retry("podman pull $alpine", delay => 15, retry => 6);
```

Comment:
IMO i do not know why do we need this exactly. 
- adds delay. alpine is really small anyway
- reduce coverage of the `run` capabilities, especially in `engines_and_tools`
- using our private *registry* the most of the problems should be resolved regarding pulling. Otherwise we should look at the registry itself.

Again i am not sure what/or how much improvement will be. However if you have concerns i would recommend to run a few stress tests without this line and see it you will find any.

Reply:
> * adds delay. alpine is really small anyway
To see my 'concern' please see the error linked in the description.

> * reduce coverage of the `run` capabilities, especially in `engines_and_tools`
I understand that. It is questionable.

> * using our private _registry_ the most of the problems should be resolved regarding pulling. Otherwise we should look at the registry itself.
Yes, this will already improve the situation.

Let's talk about this tomorrow.

Reply:
The problem is not the image size, the problem are often network infra problems. Therefore I'd also increase the delay actually to at least 30 seconds, otherwise we risk to see failures related to the same problem multiple times.

```suggestion
    script_retry("podman pull $alpine", delay => 30, retry => 3);
```

WDYT?

Reply:
What i suggest is of making sure that the images pulls from `18.156.2.117:5000`. and if this doesnt improve things then try something (maybe `script_retry`) in another step.

Reply:
Let's go with Yianni's suggestion :smile: Please review again.

Reply:
diki